packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -107,57 +108,142 @@ describe('Component > InteractionLayer', function () {
+            target: {
+              setPointerCapture: sinon.stub(),
+              releasePointerCapture: sinon.stub()
+            }
+          }
+          wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
+          wrapper.find(DrawingCanvas).simulate('pointermove', fakeEvent)
+          expect(mockMark.initialDrag).to.have.been.calledOnce()
+        })
+
+        it('should capture the pointer', function () {
+          const fakeEvent = {
+            pointerId: 'fakePointer',
+            type: 'pointer',
+            target: {
+              setPointerCapture: sinon.stub(),
+              releasePointerCapture: sinon.stub()
+            }
+          }
+          wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
+          wrapper.simulate('pointermove', fakeEvent)
+          expect(fakeEvent.target.setPointerCapture.withArgs('fakePointer')).to.have.been.calledOnce()
+        })
+
+        describe('with a TranscriptionLine mark already in progress', function () {
+          it('should call the handlePointerDown function once', function () {
+            const mockDrawingTask = DrawingTask.TaskModel.create({
+              activeToolIndex: 0,
+              instruction: 'draw a mark',
+              taskKey: 'T0',
+              tools: [
+                {
+                  marks: {},
+                  max: 2,
+                  toolComponent: TranscriptionLine,
+                  type: 'transcriptionLine'
+                }
+              ],
+              type: 'drawing'
+            })
+            activeTool = mockDrawingTask.activeTool
+            sinon.stub(activeTool, 'createMark').callsFake(() => mockMark)
+            sinon.stub(activeTool, 'handlePointerDown').callsFake(() => mockMark)
+
+            wrapper = shallow(
+              <InteractionLayer
+                activeMark={mockMark}
+                activeTool={activeTool}
+                frame={2}
+                height={400}
+                width={600}
+              />
+            )
+
+            const fakeEvent = {
+              pointerId: 'fakePointer',
+              type: 'pointer',
+              target: {
+                setPointerCapture: sinon.stub(),
+                releasePointerCapture: sinon.stub()
+              }
+            }
+            wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
+            wrapper.find(DrawingCanvas).simulate('pointerdown', fakeEvent)
+            expect(activeTool.handlePointerDown).to.have.been.calledOnce()
+          })
+        })
+      })
+    })
+
+    describe('onPointerUp', function () {
+      let mockedContext
+      before(function () {
+        mockedContext = sinon.stub(React, 'useContext').callsFake(() => { return { svg, getScreenCTM } })
       })
 
-      it('should drag the new mark on pointer down + move', function () {
+      after(function () {
+        mockedContext.restore()
+      })
+
+      it('should set the mark to finished', function () {
```

Comment:
I think this is true for most drawing tools but only true for transcription tasks on the second `pointerup`. That might have been what tripped up when trying to write tests before.